tests/Tests/500__jupyterhub/test-versions.robot

```diff
@@ -0,0 +1,83 @@
+*** Settings ***
+Force Tags       Sanity
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
```

Comment:
could you replace above two line with .resource file or would you like to do that during refactoring?